migrate/0006_add_min_players_game_setting.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    add_column :games, :min_players, Integer, null: false, default: 3
```

Comment:
would it make more sense to move max_players and min_players into settings?

Reply:
possibly.  Would you like this change made?

You are talking about the settings json obj that is already in the games db table?

Reply:
right, in the settings, the migration will be a bit more involved